ebuild-writing/variables/text.xml

```diff
@@ -588,5 +588,54 @@ when joining paths. For example: <c>${D%/}/</c>, <c>${ED%/}/</c>,
 </body>
 </section>
 
+<section>
+<title>Use of constant-value variables in ebuilds</title>
+<body>
+
+<p>
+Variables frequently have significant value in ebuilds, making it
+possible to avoid unnecessary repetitions and make maintenance easier.
+However, references to constant-value variables should be used with care
+as their proactive use can harm readability and increase maintenance
+burden (e.g. when renaming the package). In particular, using variables
```

Comment:
No because now you have to figure out which `${PN}` need to be replaced by the old name (i.e. can't change the value) and which need to be left as `${PN}` (i.e. are supposed to switch to the new name). When people use paths like `/usr/share/${PN}` (where the original path comes from upstream Makefile), this is PITA.